### PR TITLE
Add lint script

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: es/utils/aepp-wallet-communication/connection/browser-runtime.js [hunk 1]
```diff
@@ -55,10 +55,10 @@ function disconnect () {
  */
 function connect (onMessage, onDisconnect) {
   if (this.isConnected()) throw new Error('You already connected')
-  this.handler = ((msg, source) => {
+  this.handler = (msg, source) => {
     if (this.debug) console.log('Receive message: ', msg)
     onMessage(msg, source)
-  }).bind(this)
+  }
```
Comment (thepiwo) [inline es/utils/aepp-wallet-communication/connection/browser-runtime.js:61]: sure this is not breaking?

File: examples/browser/extension/utils/build.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-var webpack = require("webpack"),
-    config = require("../webpack.config");
+var webpack = require('webpack')
+var config = require('../webpack.config')
```
Comment (thepiwo) [inline examples/browser/extension/utils/build.js:2]: why `var` over `const`? maybe the linter should notice?

File: examples/browser/extension/utils/webserver.js [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+var WebpackDevServer = require('webpack-dev-server')
+var webpack = require('webpack')
+var config = require('../webpack.config')
+var env = require('./env')
+var path = require('path')
 
-var options = (config.chromeExtensionBoilerplate || {});
-var excludeEntriesToHotReload = (options.notHotReload || []);
+var options = (config.chromeExtensionBoilerplate || {})
+var excludeEntriesToHotReload = (options.notHotReload || [])
```
Comment (thepiwo) [inline examples/browser/extension/utils/webserver.js:8]: why `var` over `const`? maybe the linter should notice?

File: es/utils/aepp-wallet-communication/connection/browser-runtime.js (hunk 1 above)
Comment (davidyuk) [reply es/utils/aepp-wallet-communication/connection/browser-runtime.js:61]: Yes, https://stackoverflow.com/a/33308151/6176994
Also, I have checked it in the browser:
```js
> (() => this).bind({ test: true })()
Window {parent: Window, opener: null, top: Window, length: 0, frames: Window, …}
> (function() { return this }).bind({ test: true })()
{ test: true }
```
It looks like a tricky question at the interview 😅

File: examples/browser/extension/utils/build.js (hunk 2 above)
Comment (davidyuk) [reply examples/browser/extension/utils/build.js:2]: Probably the linter assumes that it is ES5 code. I would like to revisit examples later.

**Pull request merged**